src/SpectatorView.Native/SpectatorView.Compositor/Compositor/DeckLinkDevice.cpp

```diff
@@ -164,7 +164,7 @@ class OutputScheduler : public IDeckLinkVideoOutputCallback
         for (int i = 0; i < MAX_NUM_OUTPUT_FRAMES; i++)
         {
             HRESULT result = m_deckLinkOutput->CreateVideoFrame(
-                FRAME_WIDTH, FRAME_HEIGHT, FRAME_WIDTH * ((pixelFormat == BMDPixelFormat::bmdFormat8BitYUV) ? FRAME_BPP_RAW : FRAME_BPP),
+                FRAME_WIDTH, FRAME_HEIGHT, FRAME_WIDTH * ((pixelFormat == BMDPixelFormat::bmdFormat8BitYUV) ? FRAME_BPP_YUV : FRAME_BPP_RGBA),
```

Comment:
>FRAME_BPP_YUV [](start = 110, length = 13)

Is there a reason these values changed?

Reply:
I renamed "FRAME_BPP_RAW" to "FRAME_BPP_YUV" and "FRAME_BPP" to "FRAME_BPP_RGBA". It was confusing to name it "raw" when that's not a pixel format, and the introduction of a constant for NV12 made it even more confusing